### ISSUE TITLE
Automatically shutdown/reboot device by only holding the poweroff-button

### ISSUE THREAD

**Issue by Anchor5178** [GrapheneOS/os-issue-tracker#4043]
When you need to shutdown/reboot your pixel in a critial situation quickly to get it into BFU-mode, holding the poweroff-button and then finding the right button onscreen can be challenging or too slow. Just holding the poweroff-button to bring the device into BFU-mode would benefit in many situations. It is not just quicker and safer to do in a stress-situation but you are also able to shutdown the phone while for example it is only in your pocket.

This is also the behaviour on iPhones where you only have to hold the poweroff- and the volumeup-buttons for like 5 seconds.

**Comment by Anchor5178** [comment]
> I don't think this is a good idea because there is the possibility of an accidental long press (I do it on occasion).

It can be implemented with a delay of like 5 seconds. If even apple implemented this behaviour I think the general user will not do it by accident as often

**Comment by KxnielOutis** [comment]
I agree, the current behavior of holding the power button and having to tap the right button is not at all trivial in critical situations.

Holding the power button for 5 seconds (or a customizable amount) should restart the phone.
